src/Game/DudeObject.h

```diff
@@ -92,6 +96,7 @@ namespace Falltergeist
                 int _experience = 0;
                 int _statsPoints = 0;
                 int _skillsPoints = 3;
+                std::vector<unsigned> _perks;
```

Comment:
We know the count for perks. Therefore it is better to use `std::array` here as it will work faster